macros/vc.asm

```diff
@@ -0,0 +1,83 @@
+vc_hook: MACRO
+if DEF(_RED_VC) || DEF(_BLUE_VC)
```

Comment:
pokered and pokeyellow, unlike pokegold and pokecrystal, standardize on all-caps rgbds directives. So `IF` and `ENDC` and so on.